src/test/ui/issues/issue_56411.rs

```diff
@@ -0,0 +1,5 @@
+// compile-pass
+
+struct T {}
+
+fn main() {}
```

Comment:
These files usually go in an `auxiliary` directory. I'm ok with this if @petrochenkov is too.

Reply:
yes, but `// aux-build:` would allow to build it as an extern crate, not to use it as a module. And the `auxiliary` folder doesn't contain a `mod.rs` file, so you can't import a module from it...

Reply:
Could you rename the file into something less subtle than "same file, but with underscore", e.g. `issue_56411_aux.rs`?